app-engine/java/gradle/src/com/google/cloud/tools/intellij/appengine/java/gradle/AppEngineGradleModel.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.appengine.java.gradle;
+
+/** Model containing information obtained from the Gradle integration. */
+public interface AppEngineGradleModel {
```

Comment:
Do we need a separate interface for this model, are we expecting more implementations?

Reply:
probably don't need it. I'll check

Reply:
turns it out has to be an interface, if its not sync fails with:

`Cause: com.google.cloud.tools.intellij.appengine.java.gradle.AppEngineGradleModel is not an interface`